Cargo.toml

```diff
@@ -39,9 +39,9 @@ console_error_panic_hook = "0.1.7"
 futures-util = "0.3.27"
 http = "0.2.6"
 js-sys = "0.3.49"
-matrix-sdk-common = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "7440ce0a0cfecf6c5e121e019470fde6ff38fefd", features = ["js"] }
-matrix-sdk-indexeddb = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "7440ce0a0cfecf6c5e121e019470fde6ff38fefd" }
-matrix-sdk-qrcode = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "7440ce0a0cfecf6c5e121e019470fde6ff38fefd", optional = true }
```

Comment:
If you don't pin a revision, I'm worried this may try to use the latest one, and update the `Cargo.lock` for next users, so I'd recommend to keep on pinning a commit hash here (unless you've tested this and it doesn't change behavior even after new commits to upstream have landed).

Reply:
We discussed this in the sdk dev room (https://matrix.to/#/!qSsPTKDfMGYqhgiLPJ:flipdot.org/$Qd_cwgzB-POjqR7HicHup0jj-_n6GoxyTEAgmopBtkY?via=flipdot.org&via=matrix.org&via=element.io).

AIUI, `Cargo.lock` will only be updated if you do a `cargo update` or similar. I think that's fine.